### PR TITLE
Foorm: preview & style

### PR DESCRIPTION
**Pull request by breville**
This adds the ability to quickly preview a form on non-production environments, using URLs such as

https://studio.code.org/foorm/preview/surveys/pd/workshop_csf_intro_post and
https://studio.code.org/foorm/preview/surveys/pd/workshop_csf_intro_post

(The workshop used for these first two forms is just the first entry in `Pd::Workshop`.)

It also does some styling of the forms.  SurveyJS offers a few ways to do custom styling, including [programmatically providing additional CSS classes](https://surveyjs.io/Examples/Library/?id=survey-cssclasses&platform=Reactjs&theme=modern#content-js) and [programmatically adding new CSS properties](https://surveyjs.io/Examples/Library/?id=custom-theme&platform=Reactjs&theme=modern#content-js).  However, I just decided to [provide an object of override CSS classes](https://surveyjs.io/Examples/Library/?id=survey-customcss&platform=Reactjs&theme=modern#content-js).  The default object, as described [here](https://github.com/surveyjs/survey-library/blob/master/src/defaultCss/cssstandard.ts), has its default CSS classes still listed, but I've built an additional set of override classes that update the styling to match our site.  It's not an ideal system, but it fits the existing SurveyJS implementation.

# desktop

foorm-styling-0

# tablet

foorm-styling-0b

# mobile

foorm-styling-0c

### CODE REVIEW

File: dashboard/app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -3291,3 +3291,55 @@ h2.danger {
     color: $yellow;
   }
 }
+
+.foorm-reset-font {
+  font-family: inherit !important;
+}
+
+.foorm-adjust-body {
+  border: none !important;
```
Comment (molly-moen) [inline dashboard/app/assets/stylesheets/application.scss:3300]: for my information: what does !important do?

**Review by molly-moen — APPROVED** [review]
Looks good! Out of curiosity, what do matrices look like on mobile?

File: dashboard/app/assets/stylesheets/application.scss (hunk 1 above)
Comment (breville) [reply dashboard/app/assets/stylesheets/application.scss:3300]: It's not an ideal solution, but it overrides the default application order of CSS: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

**Comment by breville** [comment]
Oh, the screenshot cut off early, but the solution to matrices on narrow devices is actually a reasonable one:

foorm-styling-0d

**Pull request merged**